src/app/api/teams/route.js

```diff
@@ -72,20 +72,38 @@ export async function PUT(req) {
   const { objects, attribute, status } = await req.json();
 
   try {
-    objects.forEach(async (object) => {
-      if (attribute === "role") {
-        object.uids.forEach(async (member) => {
-          await updateDoc(doc(db, "users", member), {
-            team: deleteField(),
-          });
-        });
-        await deleteDoc(doc(db, "teams", object.uid));
-      } else if (attribute === "status") {
-        await updateDoc(doc(db, "teams", object.uid), {
-          status: status,
-        });
-      }
-    });
+    await Promise.all(
```

Comment:
wait pause dont implement the promise.all yet, thats for a later time, keep the old structure but just add the if condition that u have below for it. also remove the comments